packages/odata-service-writer/src/data/manifest.ts

```diff
@@ -541,3 +526,19 @@ export async function updateManifest(
     enhanceManifest(service, convertedManifest, webappPath, fs, forceServiceUpdate);
     fs.writeJSON(manifestPath, convertedManifest);
 }
+
+/**
+ * Trims leading and trailing slashes from the input string.
+ *
+ * @param input - the string from which leading and trailing slashes will be removed.
+ * @returns The input string with leading and trailing slashes removed.
+ */
+function trimSlashes(input: string): string {
+    while (input.startsWith('/')) {
```

Comment:
Could this be used as an attack vector?

Reply:
regex was giving following sonar issue for me:
![image](https://github.com/user-attachments/assets/e4992e8f-8baf-4820-8761-b3442dbf160f)

Reply:
Yeah I think you would need to add a character limit anyway or Sonar would complain about DOS attack. Previously we needed to limit the '/' check.
```
// We remove trailing slashes (up to 10, infinite would allow DOS attack) from the host to avoid double slashes when appending the service path.
        this._destinationUrl = servicePath
            ? destUrl.replace(
                  `https://${destination.Name.toLowerCase()}.dest`,
                  destination.Host.replace(/\/{1,10}$/, '')
              )
            : destination.Host;
```